### PR TITLE
initial PowerNV class support

### PR DESCRIPTION
**Pull request by sharkcz**
it resolves rhbz#1303219

### CODE REVIEW

**Comment by poncovka** [comment]
**Requires:** https://github.com/storaged-project/blivet/pull/782

**Comment by poncovka** [comment]
Jenkins, test this please.

**Comment by sharkcz** [comment]
solved the grub install phase by introducing a new class and making the install method a no-op, tested and works OK

File: pyanaconda/bootloader/grub2.py [hunk 1]
```diff
@@ -614,3 +614,14 @@ def write_defaults(self):
         #       PowerVM / POWER on qemu/kvm
         defaults.write("GRUB_DISABLE_OS_PROBER=true\n")
         defaults.close()
+
+class PowerNVGRUB2(GRUB2):
+    """PowerNV GRUBv2"""
+
+    #
+    # installation should be a no-op, just writing the config is sufficient for the
+    # firmware's bootloader (petitboot)
```
Comment (poncovka) [inline pyanaconda/bootloader/grub2.py]: Please, move this comment to the docstring of the `install` method.

File: pyanaconda/bootloader/grub2.py [hunk 2]
```diff
@@ -614,3 +614,12 @@ def write_defaults(self):
...
         defaults.write("GRUB_DISABLE_OS_PROBER=true\n")
         defaults.close()
+
+class PowerNVGRUB2(GRUB2):
+    """PowerNV GRUBv2"""
+
+    def install(self, args=None):
+    """installation should be a no-op, just writing the config is sufficient for the
+    firmware's bootloader (petitboot)
+    """
```
Comment (poncovka) [inline pyanaconda/bootloader/grub2.py]: Please, fix the indentation: `IndentationError: expected an indented block`

**Pull request merged**